### ISSUE TITLE
Writes to hard disk images randomly fail with cache=writeback/writethrough

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#2]
```
What steps will reproduce the problem?
1. Use cache=none
2. Boot a freedos cd image
3. During installation to a hard disk (image) the installation fails while 
copying files with a message "Drive c is not ready".

What is the expected output? What do you see instead?
Block reads/writes should succeed instead the guest OS usually fails 
unsuccessful to write.

Please use labels and text to provide additional information.

```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 23 Jun 2012 at 7:07

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
O_DIRECT and O_SYNC had wrong values.
Fix applied to version  v0.9.1b-1 HD Cache configuration option is disabled 
until issue #3 is fixed. Closing Issue.
```

Original comment by `ideal2i...@gmail.com` on 10 Sep 2012 at 11:12
- Changed state: **Fixed**
